### PR TITLE
feat(tests): set up framework for new tests

### PR DESCRIPTION
**Pull request by hackily**
- Moves tests to its own folder
  - If test csv files are needed, they can be put into this folder.
- exports all helper functions from index.js so they can be imported into `tests.js`
- Add test setup and teardown for env variable
- Add tests for getOrThrow()

Relates to #13

### CODE REVIEW

**Comment by danisyellis** [comment]
Thanks for helping to flesh out our tests!
I'll be looking at/code reviewing/ merging in PRs later this week.

File: __tests__/test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    createIdObjects,
+    fetchPageOfDataAndFilter,
+    fetchUserDataAndAddToCSV,
+    filterContributorByTime,
+    filterResponseForImportantEvents,
+    getOrThrow,
+    parseDatesFromArgv,
+} = require('../index');
+
+const env = Object.assign({}, process.env);
```
Comment (danisyellis) [inline __tests__/test.js:12]: Let's make this variable a bit more semantic - could you re-name it `envBeforeChanges` or something like that- to make it clearer what the intent of this code is?

File: __tests__/test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            TIMEZONE:'America/Los_Angeles',
+            CSV_COLUMN_NUMBER_FOR_GITHUB_ID: '0',
+            CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID: '1',
+            GITHUB_IMPORTANT_EVENTS: 'CommitCommentEvent,IssueCommentEvent,IssuesEvent,PullRequestEvent,PullRequestReviewEvent,PullRequestReviewCommentEvent',
+        }
+    });
+    it('should throw an error if the configuration does not exist in the environment', () => {
+        expect(() => getOrThrow('configurationThatDoesNotExist')).to.throw(Error);
+    })
+    it('should return the value of a configuration that exists exists', () => {
```
Comment (danisyellis) [inline __tests__/test.js:42]: small typo here - get rid of the extra `exists`

**Review by danisyellis — CHANGES_REQUESTED** [review]
These are some excellent additions to the project, thank you!
This will make it easier for people to add more tests. 
I've requested two small changes to your PR.

(BTW, I just wanted to let you know that eventually we'll be switching from chai to jest.)

**Comment by hackily** [comment]
> (BTW, I just wanted to let you know that eventually we'll be switching from chai to jest.)
Sounds like a great issue that someone can take up :)

**Review by danisyellis — APPROVED** [review]

**Pull request merged**